NuGet.Config

```diff
@@ -4,6 +4,7 @@
     <clear />
     <add key="dotnet-eng" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-eng/nuget/v3/index.json" />
     <add key="dotnet-public" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-public/nuget/v3/index.json" />
+    <add key="dotnet-libraries" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-libraries/nuget/v3/index.json" />
```

Comment:
It's typically fine to do this with source mapping, but wondering if you have any context on why this feed isn't an upstream of `dotnet-public`? I did a quick wiki search as I've never heard of `dotnet-libraries` and found nothing about it.

Reply:
Is the source that the sdk is using for this package. Latest available version in `dotnet-public` is not the one the sdk team is using. When Nigusu started working on `dotnet package search` and created the PR on the sdk side https://github.com/dotnet/sdk/pull/36289 I realized that we have different versions and should probably work with the one in sdk so we don't work with different versions. 

This new latest version has breaking changes so probably start working with these changes now.

This is also something was commented when Fernando worked on this https://github.com/NuGet/NuGet.Client/pull/4957#discussion_r1050017436

Reply:
Totally makes sense that you want to match their package version. I just wonder about the reason they have `dotnet-libraries` separate from the other feeds. Was thinking we could ask them about this to understand the purpose behind the separation, but I found 2 repos with `nuget.configs` that have both feeds side-by-side as well. Perhaps that's enough reason for us to do the same.